AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/helper/BundleHelper.java

```diff
@@ -172,6 +173,10 @@ public BundleHelper(AndroidAnnotationsEnvironment environment, TypeMirror elemen
 			if (isTypeParcelable(type)) {
 				methodNameToSave = "put" + "Parcelable";
 				methodNameToRestore = "get" + "Parcelable";
+			} else if (annotationHelper.isAnnotatedWith(type, "org.parceler.Parcel")) {
```

Comment:
I've been going with the assumption that if you annotate your bean with `@Parcel` that would mean Parceler is on the classpath.  Perhaps we should just support collections with supported types and at least one bean annotated with `@Parcel`... would that be acceptable instead of checking explicitly that Parceler is on the classpath?